api/integrations/sheetsIntegration.js

```diff
@@ -120,7 +120,7 @@ function fetchDatesFromResult(values, arrayMatches, index, yearToSave, monthToSa
     has to be made by the number of player's collumns and not the one from date */
     for (let dateIndex = 0; dateIndex < values[index + 1].length; dateIndex += 2) {
         // Date wil be null/undefined if it's the same as previous one
-        if ((values[index][dateIndex] != '') && (values[index][dateIndex] != undefined)) {
+        if ((values[index][dateIndex] != '') && (values[index][dateIndex] !== null)) {
```

Comment:
change to `!=` so undefined also returns true :)

Reply:
ops